.github/workflows/main.yml

```diff
@@ -81,7 +81,7 @@ jobs:
       if: startsWith(matrix.os, 'macos')
       run: |
         brew cleanup -q
-        brew update -q
+        # brew update -q
```

Comment:
Is this related to the badges problem, or just another unrelated fix?

Reply:
No, it's unrelated. This just fixes the CI on macOS.